src/BPSFeed.sol

```diff
@@ -35,6 +39,9 @@ contract BPSFeed is IBPSFeed, Owned {
 
     /// @inheritdoc IBPSFeed
     function updateRate(uint256 _rate) external onlyOwner {
+        if (_rate < InitialRate ||_rate > currentRate + RateUpdateCap) { 
```

Comment:
What if this method is called multiple times? Then the update cap doesn't matter